components/bluesky/actions/create-post/create-post.mjs

```
@@ -15,6 +15,40 @@
       description: "The text content of the post.",
     },
   },
+  methods: {
+    parseUrls(text) {
+      const spans = [];
+      const urlRegex = /(?:[$|\W])(https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*[-a-zA-Z0-9@%_+~#//=])?)/g;
+
+      let match;
+      while ((match = urlRegex.exec(text)) !== null) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)